src/NutritionValues/nutriments.js

```diff
@@ -0,0 +1,9 @@
+export default {
+  Energy: 'nutriment_energy',
+  Fat: 'nutriment_fat',
+  'Sat FAt': 'nutriment_saturated-fat',
```

Comment:
```suggestion
  'Sat Fat': 'nutriment_saturated-fat',
```
Not quite sure the capitalization was intentional